edweiss-app/app/(app)/lectures/slides/index.tsx

```diff
@@ -111,127 +123,98 @@ const LectureScreen: ApplicationRoute = () => {
+            {/* Buttons for page change and fullScreen toggle */}
 
-    // Landscape display for the screen
-    const setLandscape = async () => {
-        await ScreenOrientation.lockAsync(ScreenOrientation.OrientationLock.LANDSCAPE);
-        updateUI(ScreenOrientation.Orientation.LANDSCAPE_LEFT);
-    };
+            <TView flexDirection='row' justifyContent='space-between' pr={'sm'} pl={'sm'}>
+                <TTouchableOpacity backgroundColor='transparent' onPress={pageBack} pr={'md'}>
+                    <Icon size={'xl'} name='arrow-back-circle-outline' color='text'></Icon>
+                </TTouchableOpacity>
+
+                <TText>{currentPage}/{numPages}</TText>
 
-    // Display each question given as parameters as a component 
-    const renderQuestion = (question: string, key: React.Key) => (
-        <TView key={key} mb={'sm'} backgroundColor='crust' borderColor='surface0' radius={14} flex={1} flexDirection='column' ml='sm'>
-            <TText ml={16} mb={4} size={'sm'} pl={2} pt={'sm'} color='overlay2'>{t(`showtime:peer_question`)}</TText>
-            <TView pr={'sm'} pl={'md'} pb={'sm'} flexDirection='row' justifyContent='flex-start' alignItems='center'>
-                <TText ml={10} color='overlay0'>{question}</TText>
+                <TTouchableOpacity backgroundColor='transparent' onPress={pageForward} pl={'md'}>
+                    <Icon size={'xl'} name='arrow-forward-circle-outline' color='text'></Icon>
+                </TTouchableOpacity>
             </TView>
-        </TView>
+
+            <TTouchableOpacity backgroundColor='transparent' onPress={() => {
+                !isLandscape && setLandscape();
+                isLandscape && ScreenOrientation.unlockAsync();
+                setIsFullscreen(!isFullscreen);
+            }}>
+                <Icon size={'xl'} name={isFullscreen ? 'contract-outline' : 'expand-outline'} dark='text' testID='fullscreen-toggle'></Icon>
+            </TTouchableOpacity>
+        </TView >
     );
 
-    // Function for adding new question into the firebase storage
-    async function addQuestion(question: string) {
-        try {
-            await callFunction(LectureDisplay.Functions.createQuestion, {
-                courseId: courseName,
-                lectureId: lectureId,
-                question: question
-            });
-        } catch (error) {
-            console.error("Error creating question:", error);
-        }
-        setQuestion('');
-    }
+    const ContentView = (widthPercent: string, heightPercent: string) => (
+        <TView flexDirection='column' mr={'xl'} style={{ width: widthPercent as DimensionValue, height: heightPercent as DimensionValue }}>
+            <TScrollView b={'sm'} mt={25} mr={'md'} ml={'md'} radius={'lg'} flex={1}>
+                {currentLecture.audioTranscript?.[currentPage] ? (
+                    <TText pl={'sm'} pr={'sm'}>{currentLecture.audioTranscript[currentPage]}</TText>
+                ) : (
```

Comment:
This part of the code is going to evolve as I am preparing the multi language support. Do you have any other place except on this file were you access the audioTranscript ? because I might be going to change the implementation.

Reply:
No this is the only occurence of audioTranscript, maybe some tests done in my part will need to be rechecked but nothing else